.github/workflows/todo.yml

```diff
@@ -2,6 +2,19 @@ name: To Do app CI & CD
 on: [push, pull_request]
 
 jobs:
+
+  deployingtoheroku:
+    name: heroku
+    runs-on: ubuntu-latest
+    steps:
+    - run: docker login  --username kamolsavar --password ${{secrets.password}}
+    - run: docker build --target production --tag kamolsavar/heroku:latest .
+    - run: docker push kamolsavar/heroku:latest
+    - run: docker tag  kamolsavar/heroku:v1 registry.heroku.com/todo-app-london/web
```

Comment:
This will fail. You are building an image called `kamolsavar/heroku:latest` in your "docker build" step.

Here you are looking for an image called `kamolsavar/heroku:v1`.

Reply:
Got spot
Changes it latest now